uppy/src/main/java/com/everli/uppy/Uppy.kt

```diff
@@ -32,18 +32,32 @@ object Uppy : UppySdk {
     private lateinit var uppyService: UppyService
 
     private var deviceId: String? = null
-
-    fun init(serverUrl: String, slug: String, deviceId: String? = null) {
+    private var forcedTitle: Int? = null
+    private var title: Int? = null
+    private var message: Int? = null
+
+    fun init(
```

Comment:
Can we please avoid needed to pass also these strings as arguments?

I do like the way [Applivery](https://github.com/applivery/applivery-android-sdk) does, which basically looks for those strings inside the host apks.

Reply:
You're 100% right!!! This is way cleaner and simpler!!! I like it very much, thanks 😄